openldap/README.md

```diff
@@ -76,30 +74,41 @@ Add this configuration block to your `openldap.yaml` file to start gathering you
   instances:
       - url: ldaps://localhost
         port: 686
-        username: <your monitor user DN>
-        password: <your monitor user password>
+        username: <USER_DISTINGUISHED_NAME>
+        password: <PASSWORD>
 ```
 
 See the [sample openldap.yaml][2] for all available configuration options.
 
 [Restart the Agent][3] to begin sending OpenLDAP metrics to Datadog.
 
-### Validation
+#### Log Collection
 
-[Run the Agent's `status` subcommand][4] and look for `openldap` under the Checks section:
+**Available for Agent >6.0**
 
-```
-  Checks
-  ======
-    [...]
+1. Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:
+
+    ```
+    logs_enabled: true
+    ```
 
-    openldap
-    --------
-      - instance #0 [OK]
-      - Collected 26 metrics, 0 events & 1 service check
+2. Add this configuration block to your `openldap.d/conf.yaml` file to start collecting your Openldap logs:
 
-    [...]
-```
+    ```yaml
+      logs:
```

Comment:
Nice for the yaml tag. There is an extra level of indentation that you can remove though.

Reply:
The extra level was asked by docs in a previous PR https://github.com/DataDog/integrations-core/pull/3995#discussion_r298292439